src/ui/toCSSLineSpacing.js

```diff
@@ -1,17 +1,19 @@
 // @flow
 
-export const SINGLE_LINE_SPACING = '125%';
-export const DOUBLE_LINE_SPACING = '200%';
+// Line spacing names and their values.
+export const LINE_SPACING_100 = '125%';
+export const LINE_SPACING_115 = '138%';
+export const LINE_SPACING_150 = '165%';
+export const LINE_SPACING_200 = '232%';
 
-// In Google Doc, "single line height" is exported as style
-// "line-height: 1.15" which is too narrow to read.
-// This defines the deprecate line spacing values that should me migrated.
-const DEPRECATED_SINGLE_LINE_SPACING_VALUES = new Set(['115%', '100%']);
+export const SINGLE_LINE_SPACING = LINE_SPACING_100;
```

Comment:
do we need to export both `LINE_SPACING_100 / 200` and `SINGLE / DOUBLE_LINE_SPACING`?

Reply:
Not really. I keep them for backward compatibility reason.
But I can remove them if you prefer.

Reply:
may be worth removing to reduce confusion.